libevmasm/KnownState.h

```diff
@@ -29,12 +29,18 @@
 #include <tuple>
 #include <memory>
 #include <ostream>
-#pragma warning(push)
-#pragma GCC diagnostic push
+
+#if defined(__clang__)
+#pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wredeclared-class-member"
```

Comment:
Do we still need this in the first place? Wouldn't it make sense fixing the code instead?